src/odemis/driver/autoscript_client.py

```diff
@@ -0,0 +1,2161 @@
+            self.server._pyroClaimOwnership()
+            return self.server.create_line(parameters)
+
+    def create_circle(self, parameters: dict) -> dict:
+        """
+        Create a circle milling pattern.
+        :param parameters: (dict) Dictionary containing the pattern parameters.
+        :return: (dict) Dictionary containing the pattern parameters.
+        """
+        with self._proxy_access:
+            self.server._pyroClaimOwnership()
+            return self.server.create_circle(parameters)
+
+    def start_milling(self) -> None:
+        """Start the milling. This is non-blocking"""
+        with self._proxy_access:
+            self.server._pyroClaimOwnership()
+            self.server.start_milling()
+
+    def run_milling(self) -> None:
+        """Run the milling. This is blocking."""
+        with self._proxy_access:
+            self.server._pyroClaimOwnership()
+            self.server.run_milling()
+
+    def pause_milling(self) -> None:
+        """Pause the milling. This can be resumed."""
+        with self._proxy_access:
+            self.server._pyroClaimOwnership()
+            self.server.pause_milling()
+
+    def stop_milling(self) -> None:
+        """Stop the milling. This cannot be resumed, must be re-started."""
+        with self._proxy_access:
+            self.server._pyroClaimOwnership()
+            self.server.stop_milling()
+
+    def resume_milling(self) -> None:
+        """Resume the milling."""
+        with self._proxy_access:
+            self.server._pyroClaimOwnership()
+            self.server.resume_milling()
+
+    def get_patterning_state(self) -> str:
+        """Returns: (str) the state of the patterning."""
+        with self._proxy_access:
+            self.server._pyroClaimOwnership()
+            return self.server.get_patterning_state()
+
+    def get_patterning_mode(self) -> str:
+        """Returns: (str) the mode of the patterning."""
+        with self._proxy_access:
+            self.server._pyroClaimOwnership()
+            return self.server.get_patterning_mode()
+
+    def set_patterning_mode(self, mode: str) -> None:
+        """Set the mode of the patterning. (Serial, Parallel)"""
+        with self._proxy_access:
+            self.server._pyroClaimOwnership()
+            self.server.set_patterning_mode(mode)
+
+    def clear_patterns(self) -> None:
+        """Clear all patterns."""
+        with self._proxy_access:
+            self.server._pyroClaimOwnership()
+            self.server.clear_patterns()
+
+    def set_default_application_file(self, application_file: str = "Si") -> None:
+        """
+        Set the default application file.
+        :param application_file: (str) Name of the default application file.
+        """
+        with self._proxy_access:
+            self.server._pyroClaimOwnership()
+            self.server.set_default_application_file(application_file)
+
+    def set_default_patterning_beam_type(self, channel: str) -> None:
```

Comment:
Is the arg for this function correct? I had expect `patterning_beam_type`

Reply:
channel is the same as beam_type (electron or ion)